src/components/cc-header-orga/cc-header-orga.js

```diff
@@ -66,7 +66,7 @@ export class CcHeaderOrga extends LitElement {
         ` : ''}
 
         ${!this.error ? html`
-          <cc-img class="logo" ?skeleton=${skeleton} src=${ifDefined(orga.avatar)} text=${initials}></cc-img>
+          <cc-img class="logo" ?skeleton=${skeleton} src=${ifDefined(orga.avatar)} accessibleName=${initials}></cc-img>
```

Comment:
**nitpick:** when we're dealing with `string` and other simple types, we tend to use attributes instead of properties (same for all other occurrrences)
```suggestion
          <cc-img class="logo" ?skeleton=${skeleton} src=${ifDefined(orga.avatar)} accessible-name=${initials}></cc-img>
```